handbook/engineering/search/product.md

```diff
@@ -50,7 +50,7 @@ Iterations start every other Monday.
 
 #### Planning artifacts
 
-We use two main artifacts to plan iterations: our [planning document](https://docs.google.com/document/d/1swnkQwd724IB_HP3_Mw3KkFnfl45t-PmWcNo_1gtkQM/edit#), and a GitHub project specific to the iteration. We think of their relative roles as follows:
+We use two main artifacts to plan iterations: our [planning document](https://docs.google.com/document/d/1swnkQwd724IB_HP3_Mw3KkFnfl45t-PmWcNo_1gtkQM/edit#), and a [GitHub project](https://github.com/orgs/sourcegraph/projects/181). We think of their relative roles as follows:
```

Comment:
Are we still using the planning document? Maybe we should remove this link.

Reply:
(did not realize I had forgotten to merge this) good call, done!